sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -1353,19 +1370,17 @@ object CodeGenerator extends Logging {
             byteCodeSize
           }
         }
-        Some(stats)
+        (classCodeSize, methodCodeSizes.max, constPoolSize)
```

Comment:
I'm curious: now that we've got a nice new `ByteCodeStats` type, why use a tuple here?